### PR TITLE
readme updates

### PR DESCRIPTION
**Pull request by NickPapacostas**
lessons from installing the grid with Pascal

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,8 +86,6 @@ Start Elasticsearch from the `elasticsearch` directory:
 
 First you need to create some dev credentials in AWS - ask your friendly system administrator.
 
-Setup your awscli with a new profile `aws configure --profile media-service`.
```
Comment (NickPapacostas) [inline README.md]: this is now installed by default

File: README.md [hunk 2]
```diff
@@ -40,7 +40,6 @@ You will need to install:
 * [jq](https://stedolan.github.io/jq/)
 * [exiftool](http://www.sno.phy.queensu.ca/~phil/exiftool/)
 
-If you're using OSX, you'll also need md5 `brew install md5`.
```
Comment (NickPapacostas) [inline README.md]: this is now installed by default on osx

Comment (akash1810) [inline README.md]: even on non guardian builds?

Comment (NickPapacostas) [inline README.md]: yep, Pascal already had it installed on his when he got it ^_^

**Pull request merged**